backend/db/mongo.go

```diff
@@ -39,6 +42,34 @@ func (c *Database) insertArticle(article Article) (string, error) {
 // 	return User{}, nil
 // }
 
+// fetch all Article from mongo
+func (c *Database) FindAllArticles() (int64, []Article, error) {
+	collection := c.database.Collection("articles")
+	cursor, err := collection.Find(context.TODO(), bson.D{})
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer cursor.Close(context.TODO())
+
+	count, err := collection.CountDocuments(context.TODO(), bson.D{})
```

Comment:
The lint error means that the `err` is not checked, so a simple `if err != nil...` should work